Documentation/media/uapi/v4l/meta-formats.rst

```diff
@@ -19,6 +19,7 @@ These formats are used for the :ref:`metadata` interface only.
 .. toctree::
```

Comment:
There's still a typo in the subject of this commit.

Reply:
Noted and will fix.

Let me know when you have gone though everything and I will push an update with all fixes.